config/foreman.hiera/scenario/foreman-proxy-content/family/RedHat-8.yaml

```diff
@@ -0,0 +1,6 @@
+foreman_proxy::plugin::pulp::pulpcore_enabled: true
+foreman_proxy::plugin::pulp::pulpcore_mirror: true
+foreman_proxy::plugin::pulp::pulpnode_enabled: false
+foreman_proxy::plugin::pulp::enabled: false
+foreman_proxy_content::proxy_pulp_isos_to_pulpcore: true
+foreman_proxy_content::proxy_pulp_yum_to_pulpcore: true
```

Comment:
Should we change the class to always proxy these if Pulp 2 is disabled? The same makes sense on EL7.

Reply:
I think we are close to that being true or already there. Let me double check on that with Katello folks.

Reply:
I think that in the current approach these values end up not being used. I had hoped that we could override defaults this way but I never ended up having the time to implement it in Kafo.

Reply:
In 3.18, these are still being used as they exist today. I think then if we want the potential to pull this proxy content for EL8 support back, we should leave this in and clean everything up for 4.0 later in the dev cycle. If we want to accept that number of changes for EL8 content proxy with EL8 is too high a cost and variance, then let's target nightly and do things right. Thoughts?

Reply:
I'm not entirely sure what you mean by that.

I'm saying that these 2 lines don't work and need to be dropped unless we modify Kafo. Right now they are just misleading because Kafo does **not** use Hiera when determining defaults. The answers file has a higher priority than this so it's never used.
```suggestion
```